react/src/index.js

```diff
@@ -250,6 +250,23 @@ class App extends Component {
       scope.setUser({ email: email });
     });
 
+    // Automatically append `se`, `customerType` and `userEmail` query params to all requests
+    // (except for requests to Sentry)
+    const nativeFetch = window.fetch;
+    window.fetch = function(...args) {
+      let url = args[0];
+      let ignore_match = url.match(/^http[s]:\/\/([^.]+\.ingest\.sentry\.io\/|localhost:9989|127.0.0.1:9989).*/);
```

Comment:
What's the significance of ignoring port 9989? Consider adding this to the comment about ignoring requests to Sentry.

Reply:
Done